frontend/app/src/components/dashboard/NetWorthChart.vue

```diff
@@ -169,10 +169,10 @@ const transformData = ({ times, data }: NetValue) => {
   times.forEach((epoch, i) => {
     const value = data[i];
 
-    if (i < times.length - 1 || value > 0) {
+    if (i < times.length - 1 || value.gt(0)) {
```

Comment:
can you also convert the props to runtime props, since you've touched this file.